hudi-common/src/main/java/org/apache/hudi/common/fs/StorageSchemes.java

```diff
@@ -25,65 +25,68 @@
  */
 public enum StorageSchemes {
   // Local filesystem
-  FILE("file", false, false),
+  FILE("file", false, false, true),
   // Hadoop File System
-  HDFS("hdfs", true, false),
+  HDFS("hdfs", true, false, true),
   // Baidu Advanced File System
-  AFS("afs", true, null),
+  AFS("afs", true, null, null),
   // Mapr File System
-  MAPRFS("maprfs", true, null),
+  MAPRFS("maprfs", true, null, null),
   // Apache Ignite FS
-  IGNITE("igfs", true, null),
+  IGNITE("igfs", true, null, null),
   // AWS S3
-  S3A("s3a", false, true), S3("s3", false, true),
+  S3A("s3a", false, true, null), S3("s3", false, true, null),
   // Google Cloud Storage
-  GCS("gs", false, true),
+  GCS("gs", false, true, null),
   // Azure WASB
-  WASB("wasb", false, null), WASBS("wasbs", false, null),
+  WASB("wasb", false, null, null), WASBS("wasbs", false, null, null),
   // Azure ADLS
-  ADL("adl", false, null),
+  ADL("adl", false, null, null),
   // Azure ADLS Gen2
-  ABFS("abfs", false, null), ABFSS("abfss", false, null),
+  ABFS("abfs", false, null, null), ABFSS("abfss", false, null, null),
   // Aliyun OSS
-  OSS("oss", false, null),
+  OSS("oss", false, null, null),
   // View FS for federated setups. If federating across cloud stores, then append support is false
-  VIEWFS("viewfs", true, null),
+  VIEWFS("viewfs", true, null, null),
```

Comment:
`viewfs` supports atomic creation
```suggestion
  VIEWFS("viewfs", true, null, true),
```